### ISSUE TITLE
[ BUG ] hosted by error

### ISSUE THREAD

**Issue by 2c2lminecraft** [ZeroDiscord/Giveaway#47]
hello, why i have set hostedby: true in config and save and restart bot but when creating giveaway it doesn't say hostedby : @user ?

**Comment by Aim2339** [comment]
Please refer to this issue https://github.com/ZeroDiscord/Giveaway/issues/46 and 

Will have to commit this to the code, will do when I get time

**Comment by 2c2lminecraft** [comment]
After i changed , it same didn’t show hosted by

**Comment by Aim2339** [comment]
Hi, can you tell me what exactly you changed?

**Comment by Aim2339** [comment]
Or try adding this code 
```js
hostedBy: client.config.hostedBy ? interaction.user : null,
```

**Comment by 2c2lminecraft** [comment]
> Or try adding this code
> 
> ```js
> hostedBy: client.config.hostedBy ? interaction.user : null,
> ```
> 
> Just let me know if it works, haven't tested it myself

do i need to change it in ultis/mesaage.js? but after I changed to you, it's still the same

**Comment by Aim2339** [comment]
Alright I have made a PR for this, please check it and see if your problem has been resolved https://github.com/ZeroDiscord/Giveaway/pull/48

**Comment by ZeroDiscord** [comment]
Fixed in repository as of latest commit #46
